javascript/hasOwnProperty.md

```diff
@@ -0,0 +1,42 @@
+hasOwnProperty is a normal Javascript function that takes a string argument. Returns a boolean indicating whether the object has the specified property.
```

Comment:
Would it be better to say that: hasOwnProperty() is a default method in the Object prototype that takes a string argument and returns a boolean...